src/install/lockfile.zig

```diff
@@ -2651,20 +2648,22 @@ pub const Package = extern struct {
                             ),
                         ),
                     );
-                    defer dependency_version.value.workspace = path;
+                    defer {
+                        dependency_version.literal = path;
+                        dependency_version.value.workspace = path;
```

Comment:
This is not the absolute path right

This is relative to the root package.json?

Could we have an assertion here that asserts its not an absolute path

Reply:
> This is relative to the root package.json?

Yes, from the line directly above this:
https://github.com/oven-sh/bun/blob/76dee9312ba5201583ba9f8232b96397fa490808/src/install/lockfile.zig#L2640-L2653

> Could we have an assertion here that asserts its not an absolute path

Does checking `path[0] != '/'` cover this or is there a special function for that?

Reply:
`std.debug.assert(!std.fs.path.isAbsolute)`